e2e/tests/node_not_ready/node_power_off.robot

```diff
@@ -0,0 +1,491 @@
+*** Settings ***
+Documentation       Test the Longhorn resillence if cluster node powering off
+
+Resource            ../../keywords/common.resource
+Resource            ../../keywords/engine.resource
+Resource            ../../keywords/replica.resource
+Resource            ../../keywords/node.resource
+Resource            ../../keywords/volume.resource
+
+Suite Setup         set_test_suite_environment
+Test Setup          set_test_environment    ${TEST NAME}
+Test Teardown       Cleanup resource and resume state
+
+
+*** Variables ***
+${Gi}=                  2**30
+${volume_size_gb}=      1
+${sleep_interval}=      300
+${volume_type}=         RWO
+
+
+*** Test Cases ***
+Node power off with replica-VA
```

Comment:
Test case names itself if quite hard to understand, Suggest to add short descriptions for each test case.

Reply:
After thinking and experimenting, I found that words cannot accurately describe the content and differences of different tests, so I used tables to express.
![image](https://github.com/longhorn/longhorn-tests/assets/17548901/7dfd81ad-68f6-4c33-9e39-a6784a80741b)